src/plugins/embeddable/public/lib/panel/panel_header/panel_options_menu.tsx

```diff
@@ -63,20 +64,23 @@ export class PanelOptionsMenu extends React.Component<PanelOptionsMenuProps, Sta
   }
 
   public render() {
-    const { isViewMode, title } = this.props;
+    const { isViewMode, title, index } = this.props;
     const enhancedAriaLabel = i18n.translate(
       'embeddableApi.panel.optionsMenu.panelOptionsButtonEnhancedAriaLabel',
       {
         defaultMessage: 'Panel options for {title}',
         values: { title },
       }
     );
-    const ariaLabelWithoutTitle = i18n.translate(
-      'embeddableApi.panel.optionsMenu.panelOptionsButtonAriaLabel',
-      {
-        defaultMessage: 'Panel options',
-      }
-    );
+    const ariaLabelWithoutTitle =
+      index === undefined
+        ? i18n.translate('embeddableApi.panel.optionsMenu.panelOptionsButtonAriaLabel', {
+            defaultMessage: 'Panel options',
+          })
+        : i18n.translate('embeddableApi.panel.optionsMenu.panelOptionsButtonAriaLabelWithIndex', {
+            defaultMessage: 'Options for panel {index}',
+            values: { index },
```

Comment:
If this is user-facing, then I think we should `index+1` to avoid 0

Reply:
Yeah, it is already, it starts from 1.